NetCord/Embed.cs

```diff
@@ -1,24 +1,81 @@
 ﻿namespace NetCord;
 
+/// <summary>
+/// Displays embedded content such as an image or URL, alongside a title and various other fields. All text in the embed must add up to under 6000 characters, and you can only have up to 10 embeds per message.
+/// </summary>
 public class Embed : IJsonModel<JsonModels.JsonEmbed>
 {
     JsonModels.JsonEmbed IJsonModel<JsonModels.JsonEmbed>.JsonModel => _jsonModel;
     private readonly JsonModels.JsonEmbed _jsonModel;
-
+    
+    /// <summary>
+    /// The text that is placed above the description, usually highlighted. Also directs to a URL if given, has a 256 character limit.
+    /// </summary>
     public string? Title => _jsonModel.Title;
+    
+    /// <summary>
+    /// The type of the embed, always set to <see cref="EmbedType.Rich"/> for webhook embeds.
+    /// </summary>
     public EmbedType? Type => _jsonModel.Type;
+    
+    /// <summary>
+    /// The part of the embed where the main text is contained, limited to 4096 characters.
+    /// </summary>
     public string? Description => _jsonModel.Description;
+    
+    /// <summary>
+    /// A link to an address of a webpage. When set, the <see cref="Title"/> becomes a clickable link, directing to the URL.
+    /// </summary>
     public string? Url => _jsonModel.Url;
+    
+    /// <summary>
+    /// Displays time in a format similar to a message timestamp. Located next to the footer.
+    /// </summary>
     public DateTimeOffset? Timestamp => _jsonModel.Timestamp;
+    
+    /// <summary>
+    /// The color of the embed’s border in an RGB format.
+    /// </summary>
     public Color? Color => _jsonModel.Color;
+    
+    /// <summary>
+    /// Text at the bottom of the embed, limited to 2048 characters.
+    /// </summary>
     public EmbedFooter? Footer { get; }
+    
+    /// <summary>
+    /// The URL of the image, a large-sized image located below the "Description" element.
+    /// </summary>
     public EmbedImage? Image { get; }
+    
+    /// <summary>
+    /// The URL of the thumbnail, a medium-sized image in the top right corner of the embed.
```

Comment:
Not sure if documenting at as an URL is a good idea, because generally an URL should be a string